tests/basic/pure_test.go

```diff
@@ -335,3 +341,116 @@ var _ = Describe("{BringUpLargePodsVerifyNoPanic}", func() {
 		AfterEachTest(contexts, testrailID, runID)
 	})
 })
+
+// This test Creates multiple FADA volume/app (nginx) - Reboots a Node while volume creation is in progress
+/*
+
+https://portworx.testrail.net/index.php?/tests/view/72615025
+
+*/
+
+var _ = Describe("{RebootNodeWhileVolCreate}", func() {
+	var contexts []*scheduler.Context
+	JustBeforeEach(func() {
+		StartTorpedoTest("RebootNodeWhileVolCreate", "Test creates multiple FADA volume and reboots a node while volume creation is in progress", nil, 72615025)
+	})
```

Comment:
Test intention is to create/destroy should run parallel to reboot.
Use go routines to deploy apps and destroy apps and reboot nodes

Reply:
Done can you check?